.github/workflows/release.yml

```diff
@@ -23,8 +23,9 @@ jobs:
       - name: Tag
         id: autotagger
         uses: butlerlogic/action-autotag@stable
```

Comment:
A version number needs to be specified here instead of stable. [Others](https://github.com/ButlerLogic/action-autotag/issues/45#issuecomment-1825726927) have said 1.1.2 worked for them.